dali/operators/reader/video_reader_op.h

```diff
@@ -130,6 +129,7 @@ class VideoReader : public DataReader<GPUBackend, SequenceWrapper> {
   virtual void ProcessVideo(TensorList<GPUBackend> &video_output,
                             TensorList<GPUBackend> &video_batch, DeviceWorkspace &ws) {
     video_output.Copy(video_batch, ws.stream());
```

Comment:
The copy does propagate the layout. I'm not against setting in explicitly, but maybe it would be good to set the layout of prefetched batch too?